From: Ben Hutchings <ben@decadent.org.uk>
Date: Fri, 30 Sep 2016 10:05:05 +0100
Subject: [PATCH 396/397] work-simple: Add missing #include <linux/export.h>
Origin: https://git.kernel.org/cgit/linux/kernel/git/rt/linux-stable-rt.git/commit?id=0315a7e9bb741fcc5b0da554d416f044e6cbb10a

work-simple.c uses EXPORT_SYMBOL_GPL(), defined in <linux/export.h>.

Link: http://lkml.kernel.org/r/20160930090504.GX10601@decadent.org.uk

Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
Signed-off-by: Steven Rostedt <rostedt@goodmis.org>
---
 kernel/work-simple.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/work-simple.c b/kernel/work-simple.c
index c996f755dba6..e57a0522573f 100644
--- a/kernel/work-simple.c
+++ b/kernel/work-simple.c
@@ -10,6 +10,7 @@
 #include <linux/kthread.h>
 #include <linux/slab.h>
 #include <linux/spinlock.h>
+#include <linux/export.h>
 
 #define SWORK_EVENT_PENDING     (1 << 0)
 
